torchvision/io/image.py

```diff
@@ -1,15 +1,15 @@
 from enum import Enum
+from warnings import warn
 
 import torch
 
-from .._internally_replaced_utils import _get_extension_path
+from ..extension import _load_library
 
 
 try:
-    lib_path = _get_extension_path("image")
-    torch.ops.load_library(lib_path)
-except (ImportError, OSError):
-    pass
+    _load_library("image")
+except (ImportError, OSError) as e:
+    warn(f"Failed to load image Python extension: {e}")
```

Comment:
Is there a reason that this was changed from `pass` to `warn`? When running `pytest` on my project, I see warnings like:
```
../.spack/.spack-env/view/lib/python3.9/site-packages/torchvision/io/image.py:11
  /Users/Adam/.spack/.spack-env/view/lib/python3.9/site-packages/torchvision/io/image.py:11: UserWarning: Failed to load image Python extension: 
    warn(f"Failed to load image Python extension: {e}")
```
It's unclear if this is a serious bug in my installation that I should fix, or something I can safely ignore. FWIW, I'm using the pillow-SIMD backend.